code/__DEFINES/dcs/signals.dm

```diff
@@ -220,6 +220,11 @@
 #define COMSIG_ITEM_SHIFTCLICKON "item_shiftclickon"					//from base of mob/ShiftClickOn(): (/atom, /mob)
 	#define COMPONENT_ITEM_CLICKON_BYPASS (1<<0)
 
+#define COMSIG_NEW_REAGENT_ADD	"new_reagent_add"					//from add_reagent(): (/datum/reagent, amount); it is sent when a reagent gets added for the first time to a holder
+
+#define COMSIG_CHEM_BOOSTER_ON_OFF "chem_booster_on_off"		//from base of on_off(): (/datum/component/chem_booster, boost_on)
```

Comment:
I dont see this listened for anywhere

Reply:
removed COMSIG_CHEM_BOOSTER_ON_OFF signal